File: web3rpc/rpc-specs/code-samples/python/klay/filter/getFilterChanges.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

quantity = "0x63000c8f11e790cb4c93b0659319a159"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_filter_changes(quantity)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/filter/getFilterChanges.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const quantity = "0x16";
  provider.klay
    .getFilterChanges(quantity, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
